jobs/uaa/spec

```diff
@@ -1014,6 +1014,7 @@ properties:
         tokenUrl: <URL to the token endpoint of the provider>
         tokenKey: <Token verification key>
         tokenKeyUrl: <URL for token verification. Will be used if tokenKey is not specified.>
+        cacheJwks: <Option to enable caching for the JWKS (verification key for validating token signatures)>
```

Comment:
The this is is only example, The values are read generic, e.g. 

https://github.com/cloudfoundry/uaa-release/blob/develop/jobs/uaa/templates/config/uaa.yml.erb#L799-L811